docs/multi-organization-guide.md

```diff
@@ -0,0 +1,338 @@
+
+The following sections are written in the context of the above described use
+case; a multi-stage landing zone.
+
+## Propagating Code Changes Between ADF Installations
+
+With multiple ADF configurations across multiple AWS Organizations there comes a
+new challenge to maintain repositories and its configurations across multiple environments.
+This applies to the following repositories:
+
+- aws-deployment-framework-bootstrap
+- aws-deployment-framework-pipelines
+- any other landing zone repository
+
+As the requirements of the bootstrapped resources and ADF configuration evolves
+they need to be updated over time. These changes must propagate from one environment
+to the next in a coordinated, controlled fashion.
+
+With an `Environment Branching` approach it is possible to build a 'hands-off'
+automated mechanism to promote from a "dev" AWS Organization environment to a
+"prod" AWS Organization environment. This means that, for example merging code
+from the "dev" branch of a repository to the "int" branch of a repository, it
+will trigger the deployment process in the "int" AWS Organization.
+
+Implementing such an approach is out of scope for this guide as it heavily
+depends on the specific source code and CI/CD tool in use.
+
+## Best Practices for Multi-Organization ADF setups
+
+If you want to run ADF in a multi-organization setup, there are various best
+practices that should be followed. When following these recommendations, the
+content of the repository `aws-deployment-framework-bootstrap` and
+`aws-deployment-framework-pipelines` should be stage agnostic. This means that
+you can copy and paste the content of any of those two repositories into any
+AWS Organization stage ("dev", "int", "prod") and ADF will behave exactly the same.
+
+### 1. Create a dedicated adfconfig.yml Per AWS Organization
+
+One challenge with synchronizing the `aws-deployment-framework-bootstrap` repository
+across AWS Organizations is that the contents of the `adfconfig.yml` configuration
+file is typically tailored to the ADF installation. The can be solved by adding a
+custom adfconfig file for the given organization.
+
+Adding a configuration file with the name pattern `adfconfig.{organization id}.yml``
+in the root of the`aws-deployment-framework-bootstrap` repository will take
+precedence over the default `adfconfig.yml` settings file for that organization.
+
+For each AWS organization used with the ADF Framework setup an additional adfconfig
+file can be defined.
+
+The following screenshot shows the ADF root directory of a multi-organization ADF
+setup with three stages ("dev", "int", "prod"). Each AWS Organization has its own
+ADF config file with environment-specific values. d
+![adf-multi-org-root-directory-screenshot](images/aws-multi-org-adf-config-multi-organization.png)
+
+## 2. Customize the ADF Config File Per AWS Organization
+
+Once a dedicated ADF config file per tenant is setup, it needs to be customized
```

Comment:
This is the first time you talk about tenants, I'm don't think this is the right word here.
Do you mean environment (`dev`, `int`, `prod`)? Or ADF instance?

```suggestion
Once a dedicated ADF config file per environment is setup, it needs to be customized
```